perma_web/api/tests/test_current_user_resource.py

```diff
@@ -34,7 +31,7 @@ def test_get_archives_json(self):
         self.successful_get(self.detail_url + 'archives/', user=self.regular_user, count=11)
 
     def test_get_folders_json(self):
-        self.successful_get(self.detail_url + 'folders/', user=self.org_user, count=1)
+        self.successful_get(self.detail_url + 'folders/', user=self.org_user, count=2)
```

Comment:
Of all the tiny things to wonder about, when reviewing this..... when/where did org_user pick up a second folder? I'm not seeing it.